src/python/include/Simplex_tree_interface.h

```diff
@@ -233,6 +234,11 @@ class Simplex_tree_interface : public Simplex_tree<SimplexTreeOptions> {
     auto boundary_srange = Base::boundary_simplex_range(bd_sh);
     return std::make_pair(boundary_srange.begin(), boundary_srange.end());
   }
+
+  std::size_t get_serialization_size() {
```

Comment:
Why not in Simplex_tree near the function serialize that is essentially unusable without it?

Reply:
Move get_serialization_size inside Simplex_tree on [c23d967](https://github.com/GUDHI/gudhi-devel/pull/810/commits/c23d96752afe47d2302b7bf340c1bed5b22d45de)